### ISSUE TITLE
Feature Request - Configure plots and dashboards via script

### ISSUE THREAD

**Issue by chrispyatt** [ewels/MegaQC#383]
Is there a way to generate favourite plots and dashboards without having to log into the website and do it manually? I.e. via a script or config?

**Comment by multimeric** [comment]
Yes so they were planned to be configurable via the REST API. Not sure if I ended up implementing that though.
